crates/cargo-gpu/src/config.rs

```diff
@@ -14,28 +14,9 @@ impl Config {
 
     /// Convert CLI args to their serde JSON representation.
     fn cli_args_to_json(env_args: Vec<String>) -> anyhow::Result<serde_json::Value> {
-        let mut cli_args_json = serde_json::to_value(crate::build::Build::parse_from(env_args))?;
-
-        // Move `/install/spirv_install` to `/install`
-        let spirv_install = cli_args_json
-            .pointer("/install/spirv_install")
-            .context("`/install/spirv_install` not found in config")?
-            .clone();
-        *cli_args_json
-            .get_mut("install")
-            .context("`/install` not found in config")? = spirv_install;
-
-        let build = cli_args_json
-            .pointer("/build_args")
-            .context("`/build_args` not found in config")?
-            .clone();
-
-        // Move `/build_args` to `/build`
-        let object = cli_args_json.as_object_mut().context("!")?;
-        object.remove("build_args");
-        object.insert("build".to_owned(), build);
-
-        Ok(cli_args_json)
+        Ok(serde_json::to_value(crate::build::Build::parse_from(
```

Comment:
Niiiice, so much simpler 🚀

Reply:
these are some of the commits I cherry-picked from my library refactor :D

Reply:
oooh :D